docs/docker.md

```diff
@@ -17,325 +17,96 @@ Once you have Docker installed, simply prepare the config file (e.g. `config.jso
 Freqtrade provides an official Docker image on [Dockerhub](https://hub.docker.com/r/freqtradeorg/freqtrade/), as well as a [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) ready for usage.
 
 !!! Note
-    The following section assumes that docker and docker-compose is installed and available to the logged in user.
-
-!!! Note
-    All below comands use relative directories and will have to be executed from the directory containing the `docker-compose.yml` file.
-
-!!! Note "Docker on Raspberry"
-    If you're running freqtrade on a Raspberry PI, you must change the image from `freqtradeorg/freqtrade:master` to `freqtradeorg/freqtrade:master_pi` or `freqtradeorg/freqtrade:develop_pi`, otherwise the image will not work.
+    - The following section assumes that `docker` and `docker-compose` are installed and available to the logged in user.
+    - All below comands use relative directories and will have to be executed from the directory containing the `docker-compose.yml` file.
+    
 
 ### Docker quick start
 
 Create a new directory and place the [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) in this directory.
 
-``` bash
-mkdir ft_userdata
-cd ft_userdata/
-# Download the docker-compose file from the repository
-curl https://raw.githubusercontent.com/freqtrade/freqtrade/develop/docker-compose.yml -o docker-compose.yml
+=== "PC/MAC/Linux"
+    ``` bash
+    mkdir ft_userdata
+    cd ft_userdata/
+    # Download the docker-compose file from the repository
+    curl https://raw.githubusercontent.com/freqtrade/freqtrade/master/docker-compose.yml -o docker-compose.yml
 
-# Pull the freqtrade image
-docker-compose pull
+    # Pull the freqtrade image
+    docker-compose pull
 
-# Create user directory structure
-docker-compose run --rm freqtrade create-userdir --userdir user_data
+    # Create user directory structure
+    docker-compose run --rm freqtrade create-userdir --userdir user_data
 
-# Create configuration - Requires answering interactive questions
-docker-compose run --rm freqtrade new-config --config user_data/config.json
-```
+    # Create configuration - Requires answering interactive questions
+    docker-compose run --rm freqtrade new-config --config user_data/config.json
+    ```
 
-The above snippet creates a new directory called "ft_userdata", downloads the latest compose file and pulls the freqtrade image.
-The last 2 steps in the snippet create the directory with user-data, as well as (interactively) the default configuration based on your selections.
+=== "RaspberryPi"
+    ``` bash
+    mkdir ft_userdata
+    cd ft_userdata/
+    # Download the docker-compose file from the repository
+    curl https://raw.githubusercontent.com/freqtrade/freqtrade/master_pi/docker-compose.yml -o docker-compose.yml
```

Comment:
This seperation for docker-compose seems irrelevant - the only change is the docker-compose url - which points to the brach master_pi - which is not a branch on the repository.

The real change for the PI is within the docker-compose image, changing 
```
    image: freqtradeorg/freqtrade:master
    # image: freqtradeorg/freqtrade:develop
```

to 
```
    image: freqtradeorg/freqtrade:master_pi
    # image: freqtradeorg/freqtrade:develop_pi
```

Maybe using a `!!! Note` below this section to point the RPI difference out would work better.

Reply:
Changed to a note as suggested.